### PR TITLE
Add more variants and symlinks

### PR DESCRIPTION
**Pull request by isuruf**
Alternative to https://github.com/conda-forge/openblas-feedstock/pull/25

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 {% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
 
 # The version should be in the form X.Y, where X denotes major infrastructural change to how blas is managed,
-# and Y denoting blas priority. Currently Y=0 not default, Y=1 default blas.
-{% set Y = 1 if variant == 'openblas' else 0 %}
+# and Y denoting blas priority. Currently Y=0 not default, Y=2 default blas.
+{% set Y = 2 if variant == 'openblas' else 0 %}
 package:
   name: blas
   version: 1.{{ Y }}
```
Comment (jakirkham) [inline recipe/meta.yaml]: We need to bump the major version if we are changing the variants that are available (and there ordering).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,9 +18,17 @@ build:
     # The "blas" package must always track a feature, else it will be
     # prioritised over all others.
     - blas_{{ variant }}
+  {% if variant == "openblas" %}
+  script:                                             # [unix]
+    - bash {{ RECIPE_DIR }}/symlink_{{ variant }}.sh  # [unix]
+  {% endif %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: As noted elsewhere, I'd rather this live in the packages implementing BLASes and LAPACKs. This makes it easier for us to fix if things change later for any one of these packages. Also it is easier to reason about what is happening.

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+language: generic
+
+os: osx
+osx_image: xcode6.4
 
 env:
   matrix:
     
+    - BLAS_VARIANT=clapack
+    - BLAS_VARIANT=lapack
```
Comment (jakirkham) [inline .travis.yml]: Do these provide BLASes or just LAPACKs?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -3,8 +3,8 @@
 {% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
 
 # The version should be in the form X.Y, where X denotes major infrastructural change to how blas is managed,
-# and Y denoting blas priority. Currently Y=0 not default, Y=1 default blas.
-{% set Y = 1 if variant == 'openblas' else 0 %}
+# and Y denoting blas priority. Currently Y=0 not default, Y=2 default blas.
+{% set Y = 2 if variant == 'openblas' else 0 %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Probably needs to go to 3 if we are adding 2 new BLASes.

File: .travis.yml (hunk 3 above)
Comment (isuruf) [inline .travis.yml]: Both

**Comment by jakirkham** [comment]
A re-rendering PR came in so I merged it. Looks like that created conflicts, but they shouldn't be hard to fix. Can probably just prefer upstream for all non-recipes files and do another re-render to fix things.

**Comment by isuruf** [comment]
Done in conda-forge/openblas-feedstock#25

**Comment by jakirkham** [comment]
Were you still wanting to pursue the other variants?

**Pull request closed without merging**